src/guide/feature_flags.md

```diff
@@ -77,6 +78,17 @@ is implemented.
 
 It is recommended that only applications make use of this feature.
 
+## `unstable-provider-api`
+
+**default**: disabled
+
+When enabled, SNAFU-generated errors will implement the
+[`std::error::Error::provide`] method, allowing data to be retrieved
+using `request_ref` and `request_value` on a [`std::error::Error`]
+trait object reference.
+
+It is recommended that only applications make use of this feature.
+
```

Comment:
Is it worth linking back to the main section in `Snafu.md` from here?

Reply:
Good idea!

Reply:
Addressed in [this force-push](https://github.com/shepmaster/snafu/compare/498da7415cf0882bc376484a4c0d1de7c89ca018..9e9b01f75b049eeb878862e5d9e325f6c71071d8)